server/routes/api/auth.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const router = express.Router();
+const bcrypt = require('bcrypt');
+
+// User Model
+const User = require('../../models/user');
+
+// POST User Login
+router.post('/login', (req, res) => {
+  User.authenticate(req.body.email, req.body.password)
```

Comment:
Although it's not necessarily needed here, it's a good practice to explicitly tell Express that we're done with this route by *returning* this promise.